packages/auth/src/db/utils.js

```diff
@@ -53,6 +54,9 @@ exports.isProdAppID = appId => {
 }
 
 function isDevApp(app) {
+  if (!app) {
```

Comment:
What's going in with this one? Can't see a usage in the pr for `isDevApp`.

Reply:
I added this because it was causing an error when `app` was null (cache sometimes returned a null value without an error) - but now I think about it its probably better to have an error here, since its not a production app or a dev app, its just nothing.

Reply:
I agree on error here, better to handle getting into this scenario from the outside